src/internal/_symIterator.js

```diff
@@ -0,0 +1 @@
+module.exports = (typeof Symbol !== 'undefined') ? Symbol('iterator') : '@@iterator';
```

Comment:
We could define `_Symbol` like so:

```javascript
module.exports = typeof Symbol !== 'undefined' ? Symbol : function(s) { return '@@' + s; };
```

Then this file could become:

```javascript
var _Symbol = require('./_Symbol');

module.exports = _Symbol('iterator');
```

I haven't checked how many places we reference `Symbol`; this may not be worthwhile.